elixir/lib/protobuf/check_customization_applied.pb.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Trento.Checks.V1.CheckCustomizationApplied.MetadataEntry do
```

Comment:
Question: 

Why do we have an extra file for ChecksCustomValue but not for MetadataEntry?

Reply:
I guess because we also have 
defmodule Trento.Checks.V1.CheckCustomizationReset.MetadataEntry, so it is better to keep them in the correct file ?